soc/intel/intel_ish/Kconfig

```diff
@@ -1,11 +1,10 @@
 # Intel ISH family configuration options
 #
-# Copyright (c) 2023 Intel Corporation
+# Copyright (c) 2023-2024 Intel Corporation
 #
 # SPDX-License-Identifier: Apache-2.0
 
 config SOC_FAMILY_INTEL_ISH
```

Comment:
move all below "select" into Kconfig.soc with "config SOC_FAMILY_INTEL_ISH".
no need to keep here.

Reply:
> move all below "select" into Kconfig.soc with "config SOC_FAMILY_INTEL_ISH". no need to keep here.

hmm, why you think it needs to move ?

other HWMv2 SoC families have selects in `Kconfig`, e.g. https://github.com/zephyrproject-rtos/zephyr/blob/collab-hwm/soc/gd_gd32/Kconfig

also note this [short explanation](https://github.com/zephyrproject-rtos/zephyr/pull/68078#discussion_r1473052683)